### ISSUE TITLE
Sync up with Labs leaders to find out weekly meeting date and time

### ISSUE THREAD

**Issue by zachlatta** [hackedu/meta#149]
Make sure to cover #148 in this.

Also make sure to gather what day of the week the meeting, the time of the meeting, and in what timezone and record it in "schools" sheet on https://clubs.hackedu.us in the "Meeting Day of the week and Time + Timezone" column.

**Comment by jonleung** [comment]
Closing because duplicate of #135

**Comment by zachlatta** [comment]
#135 was the issue for last week, this one's still valid.

Reopening.

**Comment by jonleung** [comment]
@hellyeah, I added this to the description of this issue:

> Also make sure to gather what day of the week the meeting, the time of the meeting, and in what timezone and record it in https://clubs.hackedu.us in the "Meeting Day of the week and Time + Timezone" column

**Comment by hellyeah** [comment]
@hellyeah needs to update clubs issues.

**Comment by jonleung** [comment]
@hellyeah will be adding the day of the week and time for the rest of the clubs that currently don't have a date and time specified.

**Comment by hellyeah** [comment]
Asked everyone but Abby and Leila. Mid-conversation with them and don't want to diverge into this. Neither of their clubs starts til October.

**Comment by hellyeah** [comment]
Adding data from last night to http://clubs.hackedu.us

**Comment by hellyeah** [comment]
Waiting on day of the week from Amy, time from sneha and mitalee, and both day of the week and time for Victor and Abby.

**Comment by hellyeah** [comment]
Amy still hasn't decided on day of the week.

Sneha/Mitalee still haven't decided on a time.

Victor and Abby haven't decided on day of the week or time.

Made recurring Monday events in the Club Schedule calendar (#156)  to check in and get these details from them.
